.gitlab-ci.yml

```diff
@@ -19,18 +25,74 @@ include:
     file: '.gitlab-ci-check-docker-build.yml'
 
 test:static:
+  image: golang:1.15
```

Comment:
Not a big issue, but why do we run tests in golang 1.15, but build the docker image with golang 1.14?

Reply:
I bumped the docker image to go 1.15.